app/src/main/kotlin/com/wire/android/ui/home/conversations/sendmessage/SendMessageViewModel.kt

```diff
@@ -290,22 +290,27 @@ class SendMessageViewModel @Inject constructor(
                 attachmentBundle?.run {
                     when (assetType) {
                         AttachmentType.IMAGE -> {
-                            val (imgWidth, imgHeight) = imageUtil.extractImageWidthAndHeight(
-                                kaliumFileSystem,
-                                attachmentBundle.dataPath
-                            )
-                            sendAssetMessage(
-                                conversationId = conversationId,
-                                assetDataPath = dataPath,
-                                assetName = fileName,
-                                assetWidth = imgWidth,
-                                assetHeight = imgHeight,
-                                assetDataSize = dataSize,
-                                assetMimeType = mimeType,
-                                audioLengthInMs = 0L
-                            )
-                                .handleLegalHoldFailureAfterSendingMessage(conversationId)
-                                .handleAssetContributionEvent(assetType)
+                            if (kaliumFileSystem.exists(attachmentBundle.dataPath)) {
+                                val (imgWidth, imgHeight) = imageUtil.extractImageWidthAndHeight(
+                                    kaliumFileSystem,
+                                    attachmentBundle.dataPath
+                                )
+                                sendAssetMessage(
+                                    conversationId = conversationId,
+                                    assetDataPath = dataPath,
+                                    assetName = fileName,
+                                    assetWidth = imgWidth,
+                                    assetHeight = imgHeight,
+                                    assetDataSize = dataSize,
+                                    assetMimeType = mimeType,
+                                    audioLengthInMs = 0L
+                                )
+                                    .handleLegalHoldFailureAfterSendingMessage(conversationId)
+                                    .handleAssetContributionEvent(assetType)
+                            } else {
+                                appLogger.e("There was an FileNotFoundException error while sending image asset")
+                                onSnackbarMessage(ConversationSnackbarMessages.ErrorSendingAsset)
```

Comment:
This will display error: "There was an error trying to send that file. Please check your Internet connection and try again". Which is better then a crash but doesn't give right information to the user. It points them towards Internet issue while this is file system issue. Maybe a new message with "Couldn't read file" or something. I guess a design question.

We could also use `ErrorSendingImage` as it is a little more specific for the image. - "The picture could not be sent. Please check your internet connection and try again."
```suggestion
                                onSnackbarMessage(ConversationSnackbarMessages.ErrorSendingImage)
```